style.css

```diff
@@ -109,6 +113,54 @@ body {
   background-color: #f4aa80;
 }
 
+.mobile-menu {
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 57px;
+  height: 100vh;
+  position: fixed;
+  background-color: #262626;
+  z-index: 9999;
+  display: none;
+  padding: 24px 24px;
+}
+
+.show-menu {
+  display: block;
+}
+
+.hide-menu {
+  display: none;
+}
+
+.close-icon {
+  display: flex;
+  justify-content: flex-end;
+  margin-bottom: 100px;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly it will be better to add `cursor: pointer;` is more appropriate   to point the button  :+1:

Reply:
Hey MaganaAsiati, Thank you for the tip and the review. much appreciated.